src/migrations/20210106160931-create-activity-reports.js

```diff
@@ -0,0 +1,114 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.createTable('ActivityReports', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      resourcesUsed: {
+        allowNull: true,
+        type: Sequelize.STRING,
+      },
+      additionalNotes: {
+        allowNull: true,
+        type: Sequelize.STRING,
+      },
+      numberOfParticipants: {
+        allowNull: true,
+        type: Sequelize.INTEGER,
+      },
+      deliveryMethod: {
+        allowNull: true,
+        type: Sequelize.STRING,
+      },
+      duration: {
+        allowNull: true,
+        type: Sequelize.DECIMAL(3, 1),
+      },
+      endDate: {
+        allowNull: true,
+        type: Sequelize.DATEONLY,
+      },
+      startDate: {
+        allowNull: true,
+        type: Sequelize.DATEONLY,
+      },
+      participantType: {
```

Comment:
Is this for Grantees vs. NonGrantees? I'm feeling some naming confusion over this field vs. `participants` and `numberOfParticipants`.

I wonder if we even need to store this vs figuring it out from the ActivityParticipants relationships

Reply:
Yes it is for Grant vs NonGrantee. We probably could get away with not storing it but it feels easier to store it here instead of looking at the `ActivityParticipants` relationship. Although the only time we'll need to look it up is when retrieving activity reports. Creates and updates would stay pretty much the same.

I agree with the naming confusion but couldn't come up with a bettername for `grant` or `nonGrantee`. Best I could think of (and this isn't a good name at all!) is `subjects`, which also seems a bit overloaded.